workflow/scripts/report_fusions.py

```diff
@@ -31,13 +30,16 @@
             read_limit_SF = int(columns[2])
             read_limit_FC = int(columns[3])
             if gene2 == "housekeeping":
-                housekeeping_genes[gene1] = [read_limit_SF, read_limit_FC]
+                housekeeping_genes[gene1] = {}
+                housekeeping_genes[gene1]["housekeeping"] = [read_limit_SF, read_limit_FC]
```

Comment:
Why add 'housekeeping' key?

Reply:
Due to having it later in the code. Now removed the key everywhere.